behave/configuration.py

```diff
@@ -452,14 +487,80 @@ def read_configuration(path):
     return result
 
 
+def read_toml(path):
+    """Read configuration from pyproject.toml file.
```

Comment:
I think first line comment one line below is more compliant.
```suggestion
    """
    Read configuration from pyproject.toml file.
```